.travis.yml

```diff
@@ -22,6 +22,10 @@ matrix:
       env:
         - VARNISH_VERSION=4.1
         - DOCCHECK=true
+    - php: 7.1
+      env: SYMFONY_VERSION=^3.4 VARNISH_VERSION=3.0 COMPOSER_FLAGS="--prefer-lowest"
+    - php: 7.1
+      env: SYMFONY_VERSION=^4.0 VARNISH_VERSION=3.0 COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
This variable is not used in the `composer update` statement below, so Symfony 4 is never installed.